tutorials/ten_minutes.ipynb

```diff
@@ -10,9 +10,9 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        """
</pre><p>I think having full docstring gives a lot of overhead to go through. I think just the description should be good without args and returns</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>This makes sense. Will implement now.</p>